.github/workflows/label_pr_on_title.yml

```diff
@@ -6,14 +6,10 @@ on:
     types:
       - completed
 
-permissions:
-  contents: read
-
```

Comment:
If you remove this here, you will not be able to invoke the `reusable_export_pr_details.yml` workflow because it expects `contents: read` at the global level. So, you need to keep it here OR remove `contents: read ` in the `reusable_export_pr_details.yml` workflow and scope down the permissions.

BTW, I didn't have any alerts keeping this in Python.